app/assets/stylesheets/custom.css.scss

```diff
@@ -895,6 +895,19 @@ $grayMediumLight: #eaeaea;
   /*float: right;*/
 }
 
+.rounded-box {
+  border: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.